content/cybersecurity/concepts/data-exfiltration/data-exfiltration.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: 'Data Exfiltration'
+Description: 'Data exfiltration is the theft or unauthorized removal or movement of any data from a device.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Computer Science'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Data exfiltration** is the theft or unauthorized removal or movement of any data from a device. It usually involves an attacker stealing information from personal or corporate devices through various attack methods.
+
+## Data Exfiltration In-Use:
+Common data exfiltration attacks include social engineering and phishing campaigns. Malware is often unknowingly downloaded onto a user's device to steal login credentials. Data exfiltration may also occur when an insider moves data outside the network. This can be done by emailing it to a non-corporate email address. Although there might not be any malicious intent, the data is now at risk as it no longer can be monitored by the security team.
+
+## Types of Data Exfiltration:
+Data exfiltration can occur in various ways. Malware and insider threats are two of the more common ways it can occur. Cybercriminals will try to insert malware onto a network device which will then scan other devices on the network for sensitive information. The malware may remain latent to avoid detection while still exfiltrating data. In an insider threat scenario, trusted employees can inflict harm on an organization for their own gain in order to exfiltrate data. They can steal data by making google drive links public, moving critical files to personal devices, or abusing access privileges. 
```

Comment:
```suggestion
Data exfiltration can occur in various ways. Malware and insider threats are two of the more common ways it can occur. Cybercriminals will try to insert malware onto a network device which will then scan other devices on the network for sensitive information. The malware may remain latent to avoid detection while still exfiltrating data. In an insider threat scenario, trusted employees exfiltrate data for their own gain. This can result in serious harms to the company and may put many users at risk. They can steal data by making Google Drive links public, moving critical files to personal devices, or abusing access privileges. 
```